extensions/amp-subscriptions/0.1/platform-store.js

```diff
@@ -44,6 +49,50 @@ export class EntitlementStore {
 
   }
 
+  /**
+   * Resolves a platform in the store
+   * @param {string} serviceId
+   * @param {!./subscription-platform.SubscriptionPlatform} platform
+   */
+  resolvePlatform(serviceId, platform) {
+    this.subscriptionPlatforms_[serviceId] = platform;
+  }
+
+  /**
+   * Returns the platform for the given id
+   * @param {string} servideId
+   * @returns {!./subscription-platform.SubscriptionPlatform}
+   */
+  getPlatform(servideId) {
```

Comment:
This method seems a bit dangerous. It looks like it's fine for `local` platform, but not for others. I recommend you make it private. Eventually if we need something like this for non-local platforms, we'll add a new method with promise.